code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -310,3 +316,35 @@
 // Called in life() when the mob has no client.
 /datum/species/proc/handle_npc(var/mob/living/carbon/human/H)
 	return
+
+/datum/species/proc/handle_sprint_cost(var/mob/living/carbon/human/H, var/cost)
+	cost *= H.sprint_cost_factor
+	if (H.stamina == -1)
+		log_debug("Error: Species with special sprint mechanics has not overridden cost function.")
```

Comment:
yes it's good to keep, thats why i switched it to log debug. i use world << while testing